### DISCUSSION TITLE
Prośba o dodanie przeglądarki LibreWolf

### DISCUSSION THREAD

**Discussion by ghost** [FiltersHeroes/PolishCookieConsent#73] [Pomysły]
https://github.com/PolishFiltersTeam/PolishCookieConsent#jak-zainstalowa%C4%87 https://librewolf-community.gitlab.io/

**Comment by krystian3w** [comment]
W sensie do sekcji `Firefox/Waterfox/LibreWolf`?

**Comment by hawkeye116477** [comment]
Dla użytkowników tej przeglądarki raczej jest oczywiste, że wszystko z AMO będzie dla nich działać. To nie jest typowy fork.
